test/unit/event-handler-on-test.ts

```diff
@@ -24,17 +24,15 @@ test("receiver.on with invalid event name", () => {
 });
 
 test("receiver.on with event name of '*' throws an error", () => {
-  expect(() =>
-    receiverOn(state, "*" as EmitterWebhookEventName, noop)
-  ).toThrowError(
+  // @ts-expect-error
+  expect(() => receiverOn(state, "*", noop)).toThrowError(
     'Using the "*" event with the regular Webhooks.on() function is not supported. Please use the Webhooks.onAny() method instead'
   );
 });
 
 test("receiver.on with event name of 'error' throws an error", () => {
-  expect(() =>
-    receiverOn(state, "error" as EmitterWebhookEventName, noop)
-  ).toThrowError(
+  // @ts-expect-error
+  expect(() => receiverOn(state, "error", noop)).toThrowError(
```

Comment:
```suggestion
  expect(() => receiverOn(state, "error", noop)).toThrow(
```

[`toThrowError` is an alias](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-alias-methods.md), which we'd ideally like to remove at some point.